templates/includes/sidebar.html

```diff
@@ -0,0 +1,14 @@
+ <ul class="sidebar-nav">
+  <li>
+    <a href="{% url 'taxi:index' %}">Home page</a>
+  </li>
+  <li>
+    <a href="#">Manufacturers</a>

```

Comment:
The `href` attribute is set to '#', which means it does not link to an actual page. Please update this link to point to the actual URL for the 'Manufacturers' page.